spec/unit/queryFind.spec.ts

```diff
@@ -297,4 +297,21 @@ describe('Query.Find', function ()
 
         assert.isTrue(called);
     });
-});
\ No newline at end of file
+
+    it('list(projections).from.by.filter.limit.using.order.resume.stronglyConsistent()', async ()=>
```

Comment:
Could you add one more test for 'list.from.srtonglyConsistent()?

Reply:
Added more queryFind tests